CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.7.0`
+
+- Disallow multiple profiles with same name but different capitalizations. Thanks @katelynienaber
+- Improvements for Optional Credentials. Thanks @crawr @jellypuno
```

Comment:
Can we indicate the specific improvements?

Reply:
Do you want it in the changelog or in the readMe? It is mostly architecture.

Reply:
It's for the ReadMe but it would be nice to have a more precise note in Changelog, too.

Reply:
Just going to drop this link here (hoping that one day we can do something similar) :yum:
https://www.conventionalcommits.org/en/v1.0.0/